geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/BinnedArray.scala

```diff
@@ -131,13 +132,17 @@ abstract class WholeNumberBinnedArray[T](length: Int, bounds: (T, T)) extends Bi
   private val min = convertToLong(bounds._1)
   private val max = convertToLong(bounds._2)
 
-  require(min < max,
-    s"Upper bound must be greater than lower bound: lower='${bounds._1}'($min) upper='${bounds._2}'($max)")
+  if(min > max) {
+    val e = new IllegalArgumentException
```

Comment:
doesn't look idiomatic...maybe just throw the error instead of logging/throwing:
```
if (min > max) {
  throw new IllegalArgumentException(msg...)
}
```

and make the call handle it...else do:
```
if (min > max) {
  val msg = "msg here"
  logger.error(msg)
  throw new IAE(msg)
}
```